portfolio/src/main/webapp/pages/gallery.html

```diff
@@ -0,0 +1,59 @@
+    <ul class="topnav">
+      <li><a href="../index.html">Isabel Mitre</a></li>
+      <li class="right"><a href="about.html">ABOUT ME</a></li>
+      <li class="right"><a href="projects.html">PROJECTS</a></li>
+      <li class="right"><a href="gallery.html">GALLERY</a></li>
+      <li class="right"><a href="../index.html">HOME</a></li>
+    </ul>
+    <div id="explanantion">
+        <h1 class="center">Garden Gallery</h1>
+        <p> I love my plants. Unfortunately, I am a bad plant mother and frequently forget to 
+            water my plants. Fortunately, it is the wet season here in South Florida so they're
+            getting plenty of water regardless. They're also pretty resistant. The cotton plants 
+            are probably three years old and I have some citrus trees (seen in the butterfly and 
+            chrysalis pictures) that are probably twice that. Life will find a way!
+        </p>  
+      </div> 
+    <div class="content smaller">
+      <div class="column">
+        <img class="gallery" src="../images/gallery/butterfly.jpg" alt="Zebra longwing butterfly in citrus tree."/>
+        <img class="gallery" src="../images/gallery/chrysalis.jpg" alt="Zebra longwing chrysalis in citrus tree."/>
+        <img class="gallery" src="../images/gallery/coleus1.jpg" alt="Red coleus in planter."/>
+        <img class="gallery" src="../images/gallery/coleus2.jpg" alt="Coleus with green leaves and purple veins."/>
+        <img class="gallery" src="../images/gallery/cotton1.jpg" alt="Green cotton fruit."/>
+      </div>
+      <div class="column">
+        <img class="gallery" src="../images/gallery/cotton2.jpg" alt="Green cotton fruit."/>
+        <img class="gallery" src="../images/gallery/cotton3.jpg"/ alt="Cotton plant leaves.">
+        <img class="gallery" src="../images/gallery/lemonbalm.jpg" alt="Lemon balm."/>
+        <img class="gallery" src="../images/gallery/palmtree2.jpg" alt="Palm tree against blue sky."/>
+        <img class="gallery" src="../images/gallery/pineapple.jpg" alt="Big pinapple plant in a pot."/>
+      </div>
+      <div class="column">
+        <img class="gallery" src="../images/gallery/plant.jpg" alt="Very green and contrasted plant."/>
+        <img class="gallery" src="../images/gallery/wasabi.jpg" alt="Light green wasabi coleus."/>
+        <img class="gallery" src="../images/gallery/wasabi2.jpg" alt="Light green wasabi coleus."/>
+        <img class="gallery" src="../images/gallery/wickedwitch.jpg" alt="Dark magenta and pink coleus"/>
+        <img class="gallery" src="../images/gallery/can.jpg" alt="Mint cuttings in soda can."/>
+      </div>
+    </div>  
+    <!-- The Modal -->
+    <div id="myModal" class="modal">
+    <!-- The Close Button -->
+    <span class="close" onclick="document.getElementById('myModal').style.display='none'">&times;</span>
+    <!-- Modal Content (The Image) -->
+    <img class="modal-content" id="img01">
+    <!-- Modal Caption (Image Text) -->
+  <div id="caption"></div>
```

Comment:
The comments here seem a little redundant with the class names (ie you could probably remove the comments). Up to you.